index.bs

```diff
@@ -1872,6 +1839,311 @@ partial interface MLGraphBuilder {
     </div>
 </div>
 
+### The lstm() method ### {#api-mlgraphbuilder-lstm}
+Long Short-Term Memory [[LSTM]] recurrent network uses an input, output, forget, and cell gate to compute the output state that rolls into the output across the temporal sequence of the network.
+<script type=idl>
+enum MLLstmWeightLayout {
+  "iofg", // input-output-forget-cell gate ordering
+  "ifgo"  // input-forget-cell-output gate ordering
```

Comment:
ONNX uses 'c' to denote the cell gate while PyTorch uses 'g'. The 'c' denotation sometimes confuses with the 'C' used to denote the cell *state* in the same LSTM block. To avoid this subtle but confusing aspect of the spec, I chose to follow the PyTorch notation for this. Also in all the literature about LSTM, *g* is used throughout to denote the LSTM cell gate. I have no idea why ONNX picks something different.

Reply:
Thanks for the explanation @wchao1115 . I am fine with this convention if it is widely adopted.